bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/flamingos2/Flamingos2.java

```diff
@@ -140,8 +140,16 @@ public enum Filter implements DisplayableSpType, SequenceableSpType, LoggableSpT
             }
         },
         Y("Y (1.02 um)", "Y", new Some<>(1.02),                   Ictd.track("Y")),
-        F1056("F1056 (1.056 um)", "F1056", new Some<>(1.056),     Ictd.track("F1056")),
-        F1063("F1063 (1.063 um)", "F1063", new Some<>(1.063),     Ictd.track("F1063")),
+        F1056("F1056 (1.056 um)", "F1056", new Some<>(1.056),     Ictd.track("F1056")) {
```

Comment:
@swalker2m Should I mark these as `Ictd.unavailable`?

Reply:
Yeah, I think that makes sense.